packages/apputils/src/instancetracker.ts

```diff
@@ -380,7 +386,7 @@ export class InstanceTracker<T extends Widget>
    *
    * @param widget - The widget being saved.
    */
-  save(widget: T): void {
+  async save(widget: T): Promise<void> {
```

Comment:
This is an API change. I thought it was appropriate to allow users of `InstanceTracker.save` to know when the saving has completed.